torchgeo/datasets/spacenet.py

```diff
@@ -191,49 +199,287 @@ def __getitem__(self, index: int) -> Dict[str, Tensor]:
+        * Labels - GeoJSON
+                * labels.geojson
+
+    If you use this dataset in your research, please cite the following paper:
+
+    * https://arxiv.org/abs/1807.01232
+
+    .. note::
+
+       This dataset requires the following additional library to be installed:
+
+       * `radiant-mlhub <https://pypi.org/project/radiant-mlhub/>`_ to download the
+         imagery and labels from the Radiant Earth MLHub
+
+    """
+
+    dataset_id = "spacenet1"
+    imagery = {"rgb": "RGB.tif", "8band": "8Band.tif"}
+    label_glob = "labels.geojson"
+    collection_md5_dict = {"sn1_AOI_1_RIO": "e6ea35331636fa0c036c04b3d1cbf226"}
+
+    def __init__(
+        self,
+        root: str,
+        image: str = "rgb",
+        transforms: Optional[Callable[[Dict[str, Any]], Dict[str, Any]]] = None,
+        download: bool = False,
+        api_key: Optional[str] = None,
+        checksum: bool = False,
+    ) -> None:
+        """Initialise a new SpaceNet 1 Dataset instance.
+
+        Args:
+            root: root directory where dataset can be found
+            image: image selection which must be "rgb" or "8band"
+            transforms: a function/transform that takes input sample and its target as
+                entry and returns a transformed version.
+            download: if True, download dataset and store it in the root directory.
+            api_key: a RadiantEarth MLHub API key to use for downloading the dataset
+            checksum: if True, check the MD5 of the downloaded files (may be slow)
+
+        Raises:
+            RuntimeError: if ``download=False`` but dataset is missing
+        """
+        collections = ["sn1_AOI_1_RIO"]
+        super().__init__(
+            root, image, collections, transforms, download, api_key, checksum
+        )
+
+
+class SpaceNet2(SpaceNet):
+    r"""SpaceNet 2: Building Detection v2 Dataset.
+
+    `SpaceNet 2 <https://spacenet.ai/spacenet-buildings-dataset-v2/>`_
+    is a dataset of building footprints over the cities of Las Vegas,
+    Paris, Shanghai and Khartoum.
+
+    - *Collection features*
+
+
+    +------------+---------------------+------------+------------+
+    |    AOI     | Area (km\ :sup:`2`\)| # Images   | # Buildings|
+    +============+=====================+============+============+
+    | Las Vegas  |    216              |   3850     |  151,367   |
+    +------------+---------------------+------------+------------+
+    | Paris      |    1030             |   1148     |  23,816    |
+    +------------+---------------------+------------+------------+
+    | Shanghai   |    1000             |   4582     |  92,015    |
+    +------------+---------------------+------------+------------+
+    | Khartoum   |    765              |   1012     |  35,503    |
+    +------------+---------------------+------------+------------+
+
+    - *Imagery features*
+
+    .. list-table::
+        :widths: 10 10 10 10 10
+        :header-rows: 1
+        :stub-columns: 1
```

Comment:
This formatting seems more complex than it needs to be

Reply:
Agreed. But I couldn't find another way to render the table how I wanted. Open to suggestions.